### PR TITLE
Use URI template instead of path as a part of measurement name

### PR DESCRIPTION
**Pull request by matee911**

### CODE REVIEW

File: falcon_telegraf/base.py [hunk 1]
```diff
@@ -34,7 +34,10 @@ def get_tags(self, req: falcon.Request) -> Dict[str, str]:
     def get_metric_name(self, req: falcon.Request) -> str:
         if self._metric_name:
             return self._metric_name
-        return self._metric_name_prefix + req.path
+        if not req.uri_template:
+            # https://docs.influxdata.com/influxdb/v1.6/concepts/schema_and_data_layout/#discouraged-schema-design
+            warn('URI template is available only while processing a response; going to use path')
+        return self._metric_name_prefix + (req.uri_template or req.path)
```
Comment (NicholasSimons) [inline falcon_telegraf/base.py:40]: And if req.path is not present either? We just log self._metric_name_prefix

**Comment by NicholasSimons** [comment]
+1

File: README.md [hunk 2]
```diff
@@ -32,17 +32,14 @@ Measurement names can be set when creating middleware, prefixed or autogenerated
 ## Middlewares
 ### `LogHits`
 Creates a metric named hits-path/to/endpoint with one fields called `hits` with value 1.
-### `LogHitsContextAware`
-Similar to `LogHits` but executes *after* `on-*` method was called and adds more context data. Because of this you can add custom tags.
+Executes *after* `on-*` method was called and adds context data. Because of this you can add custom tags.
```
Comment (ekamil) [inline README.md]: > was called and adds context data
The middleware adds context data?

**Review by NicholasSimons — APPROVED** [review]
Still +1.  Willing to test if necessary.

File: falcon_telegraf/__init__.py [hunk 3]
```diff
@@ -1,9 +1,7 @@
 from .log_hits import LogHits
-from .log_hits import LogHitsContextAware
 from .timer import Timer
 
 __all__ = [
     'LogHits',
-    'LogHitsContextAware',
     'Timer',
```
Comment (ekamil) [inline falcon_telegraf/__init__.py]: Maybe instead mark `LogHitsContextAware` as deprecated and (log.warn in init) and apart from that remove the body of that class. This way you provide a pain-free method of upgrade with this highly beneficial change.

**Review by ekamil — CHANGES_REQUESTED** [review]
Please clear up wording in readme and keep backwards compat. (detailed comments follow)

**Pull request merged**